conf/routes

```diff
@@ -3,6 +3,7 @@ GET    /                           @org.sagebionetworks.bridge.play.controllers.
 GET    /index.html                 @org.sagebionetworks.bridge.play.controllers.ApplicationController.loadApp
 GET    /mobile/verifyEmail.html    @org.sagebionetworks.bridge.play.controllers.ApplicationController.verifyEmail(study: String ?= "api")
 GET    /mobile/resetPassword.html  @org.sagebionetworks.bridge.play.controllers.ApplicationController.resetPassword(study: String ?= "api")
+GET    /mobile/startSession.html   @org.sagebionetworks.bridge.play.controllers.ApplicationController.startSession(study: String ?= null, email: String ?= null, token: String ?= null)
```

Comment:
No, it's here for testing purposes. The other API is the one the app should/will use when it intercepts the link.